README.md

```diff
@@ -11,18 +11,16 @@ All types are immutable and thread-safe.
 |
 [GitHub](https://github.com/typelevel/squants)
 |
-[User Forum](https://groups.google.com/forum/#!forum/squants)
```

Comment:
Forgot to mention this. The user forum is almost unused, I'd rather have people use gitter, Any opinions about this?